packages/databases/sqlite/package.mk

```diff
@@ -3,11 +3,11 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="sqlite"
-PKG_VERSION="3.34.0"
-PKG_SHA256="bf6db7fae37d51754737747aaaf413b4d6b3b5fbacd52bdb2d0d6e5b2edd9aee"
+PKG_VERSION="3.34.01"
```

Comment:
its 3.34.1

Reply:
Yes, but unhappily they have named the file 3340100 and our bash magic fails.

Reply:
there is no easy way to do it :D
maybe this way ?
```
PKG_VERSION="3.34.1"
PKG_VERSION_SQLITE="${PKG_VERSION/./}00"
PKG_URL="https://www.sqlite.org/2021/${PKG_NAME}-autoconf-${PKG_VERSION_SQLITE/./0}.tar.gz"
```

Reply:
what about https://github.com/sqlite/sqlite/releases/tag/version-3.34.1 🤔

Reply:
looks very different content wise, the tea directory does not even exist

Reply:
Done.